app/api/routes/recommendation_events.py

```
@@ -0,0 +1,126 @@
+    ActivityType,
+)
+from app.repositories.recommendation_repository import RecommendationRepository
+from app.utils.mongodb import serialize_object_id
+
+router = APIRouter(prefix="/events", tags=["events"])
+
+
+@router.post("", response_model=RecommendationEvent, status_code=201)
+def create_event(
+    event_data: RecommendationEventCreate,
+    db_mongo: Database = Depends(get_mongo_db),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)